dist/index.html

```diff
@@ -3,14 +3,20 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fitness</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;600;700;800&display=swap"
```

Comment:
Why did you remove integrity and crossorigin attributes?

Reply:
by mistake